packages/providers/onekey-neo-provider/src/ProviderNeoBase.ts

```
@@ -0,0 +1,17 @@
+import { IInjectedProviderNames } from '@onekeyfe/cross-inpage-provider-types';
+
+import { ProviderBase, IInpageProviderConfig } from '@onekeyfe/cross-inpage-provider-core';
+
+class ProviderNeoBase extends ProviderBase {
+  constructor(props: IInpageProviderConfig) {
+    super(props);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)